### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by CarDioLogic**
Project Finito!

### CODE REVIEW

File: CalculatorAppJMS/CalculatorAppJMS/Program.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
+﻿// See https://aka.ms/new-console-template for more information
+using CalculatorLibrary;
+
+bool endApp = false;
+Calculator calculator = new Calculator();
+int timesCalculatorUsed = 0;
+string userOption;
+bool gag = false;
```
Comment (TheCSharpAcademy) [inline CalculatorAppJMS/CalculatorAppJMS/Program.cs:8]: what does gag stand for? Could you please rename this bool with a more significant name so I know what it does? 😊

File: CalculatorAppJMS/CalculatorLibrary/CalculationsRecord.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorLibrary;
+
+public class CalculationsRecord
+{
+    public class OperationRecord //class that holds the list structure
+    {
+        public int Id { get; set; }
```
Comment (TheCSharpAcademy) [inline CalculatorAppJMS/CalculatorLibrary/CalculationsRecord.cs:13]: Could you please apply consistent casing with your properties?

File: CalculatorAppJMS/CalculatorLibrary/CalculatorLibrary.cs [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+        OperationRecord operationRecord = new OperationRecord
+        {
+            Id = operationId,
+            numA1 = num1A,
+            operation = operationA,
+            NumA2 = num2A,
+            resultA = resultA
+        };
+        operationsList.Add(operationRecord); //adds the operation record to the operationsList that we are using
```
Comment (TheCSharpAcademy) [inline CalculatorAppJMS/CalculatorLibrary/CalculatorLibrary.cs:111]: Maybe this comes from Microsoft's tutorial, but in general refrain from commenting when the code is self-explanatory, such as in line 111.

**Review by TheCSharpAcademy — CHANGES_REQUESTED** [review]
@CarDioLogic Well done in submitting another project!

I'm requested some changes so you don't move forward with bad habits. Could you please check Codacy's errors and fix them?

File: CalculatorAppJMS/CalculatorAppJMS/Program.cs (hunk 1 above)
Comment (CarDioLogic) [reply CalculatorAppJMS/CalculatorAppJMS/Program.cs]: Hi, i changed the name for something more explanatory. Its just a bool to set up a silly joke that plays out if you try to view how many times you have you used the calculator after you perform 2 operations. :)

File: CalculatorAppJMS/CalculatorLibrary/CalculationsRecord.cs (hunk 2 above)
Comment (CarDioLogic) [reply CalculatorAppJMS/CalculatorLibrary/CalculationsRecord.cs:8]: Codacy sugested some changes to the properties here. I changed them. They should all be consistent now.

File: CalculatorAppJMS/CalculatorLibrary/CalculatorLibrary.cs (hunk 3 above)
Comment (CarDioLogic) [reply CalculatorAppJMS/CalculatorLibrary/CalculatorLibrary.cs]: Deleted! Those were comments to help me understand or guide me. Forgot to delete that one.

**Pull request merged**